src/macros.rs

```diff
@@ -0,0 +1,37 @@
+//! Macros for generating boilerplate Rust smart contract code.
```

Comment:
Lets put 1) an example of the macro being used here and 2) explanations for what deploy and call do and why we use this macro.